browser/src/VariantPage/VariantPage.tsx

```diff
@@ -76,7 +86,7 @@ type VariantPageContentProps = {
     lof_curations?: any[]
     in_silico_predictors?: any[]
     non_coding_constraint: NonCodingConstraint | null
-    transcript_consequences?: any[]
+    transcript_consequences?: TranscriptConsequence[]
```

Comment:
@rileyhgrant GraphQL actually returns `null` not `undefined` for missing values like this, so `transcript_consequences: TranscriptConsequence[] | null` is what you want here, assuming that it is valid to be missing `transcript_consequences`.